### PR TITLE
READY: RTS-393

### PR DESCRIPTION
**Pull request by lukebakken**
Fix case where incoming string for Numeric does not end in ".0" to indicate floating point.

### CODE REVIEW

File: src/riak_pb_ts_codec.erl [hunk 1]
```diff
@@ -131,6 +131,16 @@ decode_row([], Acc) ->
 decode_row([#tsrow{cells = Row} | T], Acc) ->
     decode_row(T, [decode_cells(Row, []) | Acc]).
 
+-spec decode_numeric(list()) -> float().
```
Comment (alexmoore) [inline src/riak_pb_ts_codec.erl]: Spec is wrong, we take a binary here.  
This pointed out an possible bug though. Here we expect binary back from the decode operation, [but when we encode](https://github.com/basho/riak_pb/pull/135/files#diff-5a85f13716f1297dfbd5fc46fd4c48ecR109), we give it a list.  What's it supposed to be?

Comment (alexmoore) [inline src/riak_pb_ts_codec.erl]: @bkerley ^^

Comment (bkerley) [inline src/riak_pb_ts_codec.erl]: Yeahhhh…

This code (all the time series protobuffs client/server stuff, not just this PR in particular) needs tests so we can actually reason about what's going on and what works and what doesn't.

Comment (alexmoore) [inline src/riak_pb_ts_codec.erl]: Ok, I'll fix the spec for now but let's get those tests in

**Comment by alexmoore** [comment]
:+1:

**Comment by alexmoore** [comment]
+1 ef8421906a6e52837d9d645ab60dafbb198a8bf5

**Comment by alexmoore** [comment]
@borshop merge

**Pull request merged**